grails-rundeck-data-shared/src/main/groovy/rundeck/data/job/RdWorkflowStep.groovy

```diff
@@ -40,6 +40,6 @@ class RdWorkflowStep implements WorkflowStepData, PluginProviderConfiguration, V
     @Override
     @JsonIgnore
     String summarize() {
-        return "implement summarization"
+        return "Plugin["+ pluginType + ', nodeStep: '+nodeStep+']'
```

Comment:
This should match the Gorm implementation. You'll find the Gorm implementations in:
```
CommandExec.summarize()
PluginStep.summarize()
JobExec.summarize()
```
You should probably create a utility class to do this summarization rather than putting the code in this class.